### PR TITLE
Custom NavBar to suit each page on Mobile

### PR DESCRIPTION
**Pull request by ngyngcphu**
I create `NavBarController` component and `sidebar` state. Each screen has an extra navbar (ex: `AllProjectsNavBar` in screen/tickflow).

### CODE REVIEW

File: src/components/NavBarController.tsx
Comment (phucvinh57) [inline src/components/NavBarController.tsx:1]: Should we remove `screens` folder @ngyngcphu ?

File: src/utils/colorSelect.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export const selectColorStatus = (status: string) => {
```
Comment (phucvinh57) [inline src/utils/colorSelect.ts:1]: Consider to use object instead of function

**Review by phucvinh57 — COMMENTED** [review]
LGTM

File: src/components/NavBarController.tsx
Comment (ngyngcphu) [reply src/components/NavBarController.tsx:1]: I don't think so, it's still ok.

Comment (phucvinh57) [reply src/components/NavBarController.tsx:1]: Seem like scope-based component separation (`@components/tickfund`, `@components/tickflow` ...) is more suitable and clean enough. We can remove `screens` folder to reduce source code complexity.

Help me do this @ngyngcphu .

Comment (ngyngcphu) [reply src/components/NavBarController.tsx:1]: Yeah, I see.

**Pull request merged**